### ISSUE TITLE
Make sure classes do not clash with OAuth1 plugin.

### ISSUE THREAD

**Issue by justingreerbbi** [justingreerbbi/WP-REST-API-OAuth-2.0-Server#11]
Ensure that classes are renamed appropriately and that credit is left in each file. If no created is in the original, we need to ensure that we add it in.

OAuth1 and OAuth2 should be able to run side by side.

**Comment by apkoponen** [comment]
Yup. WP_REST_Client in my branch was left as is because I wasn't sure if the OAuth1 coders meant it to be used as we are using, so it would make sense to share between the plugins it as is. I'll rename it.

**Comment by apkoponen** [comment]
In addition, I run the OAuth1 plugin enabled all the time to ensure that there are no clashes.

**Comment by justingreerbbi** [comment]
Cool, that makes sense.
